website/content/docs/release-notes/consul-k8s/v1_0_x.mdx

```diff
@@ -0,0 +1,58 @@
+---
+layout: docs
+page_title: 1.0.x
+description: >-
+  Consul on Kubernetes release notes for version 1.0.x
+---
+
+# Consul on Kubernetes 1.0
+
+## Release Highlights
+
+- ** Simplified Service Mesh Deployments with Consul Dataplane **: Consul client agents are no longer deployed by default, and Consul service mesh no longer uses Consul clients to operate. A new component `consul-dataplane` is now injected as a sidecar-proxy instead of plain Envoy. `consul-dataplane` manages the Envoy proxy process and proxies xDS requests from Envoy to Consul servers. All service-mesh consul-k8s components are configured to talk directly to Consul servers
```

Comment:
```suggestion
- ** Simplified Service Mesh Deployments with Consul Dataplane:** Consul client agents are no longer deployed by default, and Consul service mesh no longer uses Consul clients to operate. A new component `consul-dataplane` is now injected as a sidecar-proxy instead of plain Envoy. `consul-dataplane` manages the Envoy proxy process and proxies xDS requests from Envoy to Consul servers. All service-mesh consul-k8s components are configured to talk directly to Consul servers.
```